src/libraries/Common/tests/Extensions/TestingUtils/Microsoft.AspNetCore.Testing/test/AssemblyFixtureTest.cs

```diff
@@ -19,7 +19,7 @@ public AssemblyFixtureTest(TestAssemblyFixture assemblyFixture, TestCollectionFi
 
         public TestAssemblyFixture AssemblyFixture { get; }
         public TestCollectionFixture CollectionFixture { get; }
-
+#if !SKIP_FAILING
```

Comment:
AssemblyFixtureTests seems unrelated to DI.tests. I think this class will be deleted eventually.

Reply:
If that's the case, then don't include the source file.